tests/testthat/_snaps/rd-usage.md

```diff
@@ -27,8 +27,8 @@
       x,
       y,
       xy = "abcdef",
-      xyz = c(`word word word word` = "abcdef", `word word word` = "abcdef",
-        `word word word` = "abcdef", `word word word` = "abcdef")
+      xyz = c(`word word word word` = "abcdef", `word word word` = "abcdef", `word word word`
```

Comment:
This only looked good by coincidence before, because the C++ wrapping code counts bytes, not characters, and `\u{A0}` is two bytes.